torch/csrc/jit/serialization/callstack_debug_info_serialization.cpp

```diff
@@ -40,6 +46,16 @@ c10::IValue InlinedCallStackSerializer::serialize(
   } else {
     elements.emplace_back(c10::IValue());
   }
+  if (cs_ptr->function()) {
+    elements.emplace_back(cs_ptr->function()->name());
+  } else {
+    auto fn_name = cs_ptr->function_name();
+    if (!fn_name.empty()) {
```

Comment:
Curious, aren't we already making sure that this is always populated (with FunctionName_UNKNOWN in the worst case)?